alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/cognitiveagents/characteristics/individual/Age.kt

```diff
@@ -12,10 +8,23 @@ enum class Age : IndividualCharacteristic {
 
     companion object {
 
+        private const val CHILD_THRESHOLD = 18
+        private const val ADULT_THRESHOLD = 60
+        private val CHILD_KEYWORDS = setOf("child", "CHILD")
+        private val ADULT_KEYWORDS = setOf("adult", "ADULT")
+        private val ELDERLY_KEYWORDS = setOf("elderly", "ELDERLY")
+
         fun getCategory(age: Int): Age = when {
```

Comment:
I suggest renaming this in `fromYears`. Looks much better: `Age.fromYears(2)`

Reply:
I agree